src/main/resources/data.sql

```diff
@@ -47,9 +48,10 @@ INSERT INTO program (
     'Program 1.1',
     (SELECT id FROM organization WHERE name = 'Org 1'),
     1999,
-    '123 Peachtree St.',
+    '1006 North Highland Avenue Northeast',
```

Comment:
I went ahead and put in real addresses for our dummy data so it will make integration with the map easier.

I chose Taco Mac locations :)